contrib/legal/privacy.html.sample

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Privacy Policy</title>
+ </head>
+
+ <body>
+  <h1>Privacy Policy</h1>
+
+  <i>Last updated: December 28, 2019</i>
+
+  <h2>Who We Are?</h2>
+
+  <p>Your Gitea Instance</p>
+
+  <h2>What Personal Data We Collect?</h2>
+
+  <p>We collect following personal data (collectively called User Personal Information):</p>
+
+  <ol>
+   <li>Registration information (username, email, password, etc.)</li>
+   <li>Profile information for your Account (such as your full name, biography, website, and location.)
```

Comment:
If we're gonna close list elements we should close them all.

```suggestion
   <li>Profile information for your Account (such as your full name, biography, website, and location.)</li>
```

Reply:
Oh, I forgot to close this `<li>` listing!